### PR TITLE
add detail about the SCRAM-SHA-512 secret that should be decoded befo…

### PR DESCRIPTION
**Pull request by Sajadorouji**
…re use.

### Type of change

_Select the type of your PR_

- Documentation

### Description

add detail about the SCRAM-SHA-512 secret that should be decoded, it's not mentioned in documentation and related to issue: https://github.com/strimzi/strimzi-kafka-operator/issues/1525

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [x] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

File: documentation/book/ref-kafka-user.adoc [hunk 1]
```diff
@@ -93,6 +93,11 @@ data:
   password: # Generated password
 ----
 
+For decode the generated password:
+----
+echo "Generated password" | base64 --decode
```
Comment (scholzj) [inline documentation/book/ref-kafka-user.adoc]:
```suggestion
echo "Z2VuZXJhdGVkcGFzc3dvcmQ=" | base64 --decode
```

File: documentation/book/ref-kafka-user.adoc [hunk 2]
```diff
@@ -93,6 +93,11 @@ data:
   password: # Generated password
```
Comment (scholzj) [inline documentation/book/ref-kafka-user.adoc]:
```suggestion
  password: Z2VuZXJhdGVkcGFzc3dvcmQ= # Generated password
```

File: documentation/book/ref-kafka-user.adoc [hunk 3]
```diff
@@ -76,7 +76,7 @@ spec:
 ----
 
 When the user is created by the User Operator, the User Operator will create a new secret with the same name as the `KafkaUser` resource.
-The secret will contain the generated password in the `password` key.
+The secret will contain the generated password in the `password` key that is encoded with base64, for using it you must decode it.
```
Comment (PaulRMellor) [inline documentation/book/ref-kafka-user.adoc]:
```suggestion
The secret contains the generated password in the `password` key, which is encoded with base64. In order to use the password it must be decoded.
```

**Comment by scholzj** [comment]
@Sajadorouji Could you please merge the suggested changes so that we can merge this PR?

**Comment by scholzj** [comment]
I fixed the comments and merged this manually. Thanks for the PR.

**Pull request closed without merging**